src/@next/DataTable/DataTable.tsx

```diff
@@ -19,8 +19,11 @@ import { TableRow } from './TableRow';
 
 export type SortDirection = 'ASCENDING' | 'DESCENDING';
 
-export type TableHeading = React.ThHTMLAttributes<HTMLTableColElement> & {
-  title?: string;
+export type TableHeading = Omit<
+  React.ThHTMLAttributes<HTMLTableColElement>,
+  'title'
+> & {
+  title?: React.ReactNode;
```

Comment:
I think it’s best to include the comment for headings prop in the `DataTableProps`. Can you mention that if the title is of ReactNode type, they must provide the id for sorting to be able to work correctly?

apart from that, can you include a console.warn with the same logic? 👆

Reply:
![image](https://github.com/glints-dev/glints-aries/assets/67429929/cc5e7265-68db-4d1f-ae1f-fcc1e25f721e)
@ninariccimarie something like this is fine for the comment?

Reply:
![image](https://github.com/glints-dev/glints-aries/assets/67429929/c5bf905e-22e5-4ce3-99f3-4f52034fa401)
and this for the console.warn

Reply:
you can wrap it in `/** comment here */` to make it show in the storybook. example: https://github.com/glints-dev/glints-aries/blob/f37a6a716932f485f9d4f717a1192599fe66d7bc/src/%40next/Tabs/Tabs.tsx#L12